### ISSUE TITLE
Color contrast does not meet WCAG standards

### ISSUE THREAD

**Issue by Otto-AA** [scttcper/ngx-toastr#929]
I think the color contrast of the notifications should be improved.

## Problem description

I've tested the default notifications on the [demo page](https://ngx-toastr.vercel.app/) and checked with the Firefox developer tools how their contrast is rated:

- Info: 3.42 (Does not meet WCAG standards for accessible text)
- Success: 3.13 (Does not meet WCAG standards for accessible text)
- Warning: 2.28 (Does not meet WCAG standards for accessible text)
- Error: meets the standard

The suggested "minimum ratio" is 4.5, so I think at least this should be the goal ([see here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/Understanding_WCAG/Perceivable/Color_contrast) for more info). Personally, I also find the warning text hard to read, even without a visual impairment.

## Suggestions

I'd suggest one of those:
- change the color defaults to have a higher contrast
- add a `goodContrast` option

**Comment by codergeek42** [comment]
I created PR 943 which could fix these info, success, and warning colors to have a minimum 4.5:1 contrast ratio. (I couldn't find an obvious way to link that patch/branch here, sorry.)

https://github.com/scttcper/ngx-toastr/pull/943
